src/cpprestsdk/cpprestsdk/Release/src/http/client/http_client_winhttp.cpp

```diff
@@ -2039,6 +2039,18 @@ class winhttp_client final : public _http_client_communicator
             case WINHTTP_CALLBACK_STATUS_SENDING_REQUEST:
             {
                 p_request_context->on_send_request_validate_cn();
+
+                try
```

Comment:
>try [](http://example.com/codeflow?start=16&length=3)

Are we directly modifying subtree modules?

Reply:
The upstream codebase is in maintenance mode.  We can try to get this change in there but it seems unlikely, and very unlikely to be able to do so in the time frame we need.